index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <script type="module" src="scripts/index.js"></script>
+    <title>Haker_news_JS</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="container__header">
```

Comment:
header could be simplified: 
`<header class="container__header">
        <img src="images/logo.gif" />
        <h1 class="logo__title">Hacker News</h1>
      </header>`

Reply:
Div logo__image can't remove. It creates border around logo.
        <div class="logo__image">
          <img src="images/logo.gif" alt="logo" />
        </div>

Reply:
Fixed